spring-webflux/src/main/java/org/springframework/web/reactive/config/PathMatchConfigurer.java

```diff
@@ -35,12 +35,10 @@ public class PathMatchConfigurer {
 	@Nullable
 	private Boolean trailingSlashMatch;
 
-
 	@Nullable
 	private Boolean caseSensitiveMatch;
 
-	@Nullable
-	private Map<String, Predicate<Class<?>>> pathPrefixes;
+	private Map<String, Predicate<Class<?>>> pathPrefixes = new LinkedHashMap<>();;
```

Comment:
Please revert all changes related to `pathPrefixes`.

The code was intentionally written without assignment, since path prefixes are optional.

The `LinkedHashMap` is lazily created on demand. Consequently, the map is only created if it's actually used.